examples/99-advanced/planets.py

```diff
@@ -0,0 +1,119 @@
+"""
+.. _planets_example:
+
+3D Earth and Celestial Bodies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Creates high-resolution renderings of the Earth and the major celestial bodies
+in our solar system for astrodynamics applications.
+This example is inspired by `planet3D-MATLAB <https://github.com/tamaskis/planet3D-MATLAB>`_ .
+
+
+Note that this example author doesn't know anything about astronomy. There may
+be inaccuracies in the representation. The purpose of this example is for
+inspiration only.
+
+"""
+import pyvista
+from pyvista import examples
+
+###############################################################################
+# Celestial bodies with the Milky Way in the background.
+
+
+# Light of the sun in the +x axis direction.
+
+light = pyvista.Light()
+light.set_direction_angle(30, -20)
```

Comment:
You are right. As you see "+x axis" view is not good because of shading. So I will change the comment.
![output](https://user-images.githubusercontent.com/7513610/188972953-d2f919eb-2271-4d00-906d-0d432f075e0e.png)

Reply:
See e4732a9fc6dd3314547e171fbd4adcc98c690da3 .